dags/ods/kktix_ticket_orders/kktix_dag.py

```diff
@@ -34,7 +34,13 @@
         provide_context=True,
     )
 
-    CREATE_TABLE_IF_NEEDED >> GET_ATTENDEE_INFOS
+    ADD_USER_TO_GATHER_TOWN_WHITELIST = PythonOperator(
+        task_id="ADD_USER_TO_GATHER_TOWN_WHITELIST",
+        python_callable=gather_town_loader.load,
+        provide_context=True,
+    )
+
+    CREATE_TABLE_IF_NEEDED >> GET_ATTENDEE_INFOS >> ADD_USER_TO_GATHER_TOWN_WHITELIST
```

Comment:
task `ADD_USER_TO_GATHER_TOWN_WHITELIST` depends on `GET_ATTENDEE_INFOS`

also, `GET_ATTENDEE_INFOS` would pass a payload via `XCOM` to `ADD_USER_TO_GATHER_TOWN_WHITELIST`